framework/wazuh/core/cluster/master.py

```diff
@@ -266,14 +267,20 @@ def process_request(self, command: bytes, data: bytes) -> Tuple[bytes, bytes]:
             logger = self.task_loggers['Agent-groups send']
             start_time = self.send_agent_groups_status['date_start']
             if isinstance(start_time, str):
-                start_time = datetime.strptime(start_time, DECIMALS_DATE_FORMAT)
+                if start_time == DEFAULT_DATE:
+                    start_time = datetime(1970, 1, 1, 0, 0, tzinfo=timezone.utc)
+                else:
+                    start_time = datetime.strptime(start_time, DECIMALS_DATE_FORMAT)
```

Comment:
It seems that the purpose of the issue has been misunderstood. What is intended is to simplify the current state of the dates in master nodes.

The cluster is a bit complex, but for example in this case, the master cannot receive the `b'syn_w_g_e` command without having previously run the `send_agent_groups_information` method. In said method, a date is already set inside `send_agent_groups_status['date_start']`, so it is not possible for the value of `start_time` to be `DEFAULT_DATE` at this point. Therefore, those `start_time == DEFAULT_DATE` conditions are not needed.

I would expect something similar to this:
```PYTHON
        elif command == b'syn_w_g_e':
            logger = self.task_loggers['Agent-groups send']
            start_time = datetime.strptime(self.send_agent_groups_status['date_start'], DECIMALS_DATE_FORMAT)
            return c_common.end_sending_agent_information(logger, start_time.timestamp(), data.decode())
```